src/emitter.ts

```diff
@@ -120,6 +120,16 @@ function isEventHandler(p: Browser.Property) {
     return typeof p["event-handler"] === "string";
 }
 
+const newToConstructorRegExp = /^new(?:<.+?>)?(\(.+?\))(?:: ?[^)]+)?$/u;
+function convertNewToConstructor(signature: string) {
```

Comment:
This seems only for manual JSON data, in that case I think those should be fixed instead.